Cargo.toml

```diff
@@ -191,3 +191,7 @@ tempfile = "3.8"
 proptest = "1.0"
 proptest-derive = "0.4"
 serial_test = "2"
+
+[patch.crates-io]
```

Comment:
Remove patch once the upgrade has been upstreamed. https://github.com/paradigmxyz/reth/pull/5032/files